content/en/data_jobs/databricks.md

```diff
@@ -57,29 +61,41 @@ You can choose to install the Agent globally, or on a specific Databricks cluste
    - Under **Init script path**, enter the path to your init script.
    - Click **Add**.
 
+#### Add your Datadog API key in Databricks
+
+1. In Databricks, on the cluster configuration page, click the **Advanced options** toggle.
+1. At the bottom of the page, go to the **Spark** tab.
+   {{< img src="data_jobs/databricks/configure-databricks-cluster-init-script.png" alt="Databricks UI, cluster configuration advanced options, Spark tab. A textbox titled 'Environment variables' contains values for DD_API_KEY and DD_SITE." style="width:100%;" >}}
+
+   In the **Environment variables** textbox, provide the values for the init script parameters.
+1. Click **Confirm**.
+
 {{% /tab %}}
 
 {{< /tabs >}}
 
-### Add your Datadog API key in Databricks
-1. Find your [Datadog API key][3].
-1. In Databricks, on the cluster configuration page, click the **Advanced options** toggle.
-1. At the bottom of the page, go to the **Spark** tab.
-   {{< img src="data_jobs/databricks/configure-databricks-spark-envvars-updated.png" alt="Databricks UI, cluster configuration advanced options, Spark tab. A textbox titled 'Environment variables' contains values for DD_API_KEY and DD_SITE." style="width:100%;" >}}
+### Init script parameters
 
-   In the **Environment variables** textbox, set values for: 
-    - `DD_API_KEY`: Your Datadog API key. (You can also use Databricks [secret management][4] to store your API key.)
-    - `DD_SITE`: Your [Datadog site][5].
+Provide the following parameters to the init script:
 
-   For example, if your Datadog site is {{< region-param key="dd_site" code="true" >}}, paste the following into the box:
+```text
+export DD_API_KEY=<YOUR API KEY>
+export DD_SITE={{< region-param key="dd_site" code="true" >}}
+export DATABRICKS_WORKSPACE=<YOUR WORKSPACE NAME>
+```
 
-   ```text
-   DD_API_KEY=<YOUR API KEY>
-   DD_SITE={{< region-param key="dd_site" code="true" >}}
-   ```
+Optionally, you can also set other init script parameters and Datadog environment variables here, such as `DD_ENV` and `DD_SERVICE`. The script can be configured using the following parameters:
```

Comment:
In the script itself, the format of the optional parameters might be a bit awkward for folks not used to bash. Maybe a formatting similar to the DD agent config yaml files can be used, where to enable the setting the user can uncomment it and set the value to true? 

```bash
## Optional parameters - boolean - default: false
## Setting any of these options to "true" will enable the respective
## telemetry to be collected
# export DD_PROCESS_AGENT_ENABLED=false
# export DD_PROFILING_ENABLED=false
# export DRIVER_LOGS_ENABLED=false
# export WORKER_LOGS_ENABLED=false
```

Reply:
Agreed, changed it to this

```bash
## Optional parameters - boolean - default: false
## Setting any of these options to "true" will enable the respective telemetry to be collected
# export DD_PROCESS_AGENT_ENABLED=false
# export DD_PROFILING_ENABLED=false
# export DRIVER_LOGS_ENABLED=false
# export WORKER_LOGS_ENABLED=false
```